attributes/config.rb

```diff
@@ -67,7 +67,7 @@
 default['nagios']['conf']['check_result_reaper_frequency']               = '10'
 default['nagios']['conf']['max_check_result_reaper_time']                = '30'
 default['nagios']['conf']['check_result_path']                           =
-  if node['platform'] == 'centos' && node['platform_version'].to_i >= 7
+  if platform?('ubuntu') && node['platform_version'].to_i >= 7
```

Comment:
Looks like some of the platform checks aren't right. This should be centos not Ubuntu. I see some other occurrences

Reply:
That's a bug that was fixed and released in Cookstyle.

Reply:
@marcosvm it looks like you may need to update your Cookstyle and run this again

Reply:
Fixed.